scripts/tests/test_validate_docstrings.py

```diff
@@ -907,12 +921,15 @@ def test_exit_status_for_validate_one(self, monkeypatch):
         exit_status = validate_docstrings.main(func_name='docstring1',
                                                prefix=None,
                                                errors=[],
-                                               output_format='default')
+                                               output_format='default',
+                                               ignore_deprecated=False)
         assert exit_status == 0
 
     def test_exit_status_errors_for_validate_all(self, monkeypatch):
         monkeypatch.setattr(
-            validate_docstrings, 'validate_all', lambda prefix: {
+            validate_docstrings, 'validate_all',
+            lambda prefix, ignore_deprecated=None:
+            {
```

Comment:
Move the curly bracket to the previous line

Reply:
did it!